### PR TITLE
Modified number 1 to number 2 inside first_file.txt

### PR DESCRIPTION
**Pull request by scononen**
This is my first pull request

### CODE REVIEW

File: first_file.txt [hunk 1]
```diff
@@ -1 +1 @@
-1
\ No newline at end of file
+2
```
Comment (scononen) [inline first_file.txt]: according hometask need to change 2 to 3

File: first_file.txt [hunk 2]
```diff
@@ -1 +1 @@
-1
\ No newline at end of file
+3
```
Comment (gh3rtzz) [inline first_file.txt]: Great. Now change the number 3 to number 4

**Review by scononen — COMMENTED** [review]
changes 3 to 4 was done

**Review by gh3rtzz — APPROVED** [review]
Approved

**Pull request merged**